deploy/template.yaml

```diff
@@ -2231,22 +2231,6 @@ Resources:
           POWERTOOLS_SERVICE_NAME: !Sub ci-scoring-${Environment}
           IPV_SESSIONS_TABLE_NAME: !Ref SessionsTable
           CLIENT_OAUTH_SESSIONS_TABLE_NAME: !Ref ClientOAuthSessionsTable
-          CI_STORAGE_GET_LAMBDA_ARN: !Sub
```

Comment:
this was done in previous PR

Reply:
Oh, I see what you mean now. Yeah, I'll merge that other PR first I think. Not 100% why this clash has happened.